pystac_client/cli.py

```diff
@@ -22,7 +22,10 @@ def search(
     """Main function for performing a search"""
 
     try:
-        result = client.search(method=method, **kwargs)
+        # the type: ignore is to silence the mypy error
+        # error: Argument 2 to "search" of "Client" has incompatible
+        # type "**Dict[str, Dict[str, Any]]"; expected "Optional[int]"  [arg-type]
```

Comment:
Can you add some extra comment text linking to https://github.com/python/mypy/issues/4441 or something similar, to prevent future us from googling to see if the issue has been fixed?